packages/storybook/stories/CallComposite/CallCompositeDocs.tsx

```diff
@@ -123,6 +124,22 @@ export const getDocs: () => JSX.Element = () => {
         useful if you want to explore the composite with multiple users.
       </Description>
 
+      <Heading>PSTN and 1:N Calling</Heading>
+      <SingleLineBetaBanner />
+      <Description>
+        The CallComposite supports making outbound PSTN and 1:N calls. 1:N is a mix between Azure Communication Users
+        and PSTN users. To make these out bound calls you need to provide a `locator` that contains participantID's that
+        you are looking to call to the [CallAdapter](./?path=/docs/composite-adapters--page). For PSTN these ID's are
+        the phone numbers that you are looking to call. For Azure Communication Users you will need to provide their
+        unique ACS aquired `userId`.
+      </Description>
+      <Description>
+        As well as these participantID's you are required to provide a [phone
+        number](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/telephony/get-phone-number?tabs=windows&pivots=platform-azcli)
```

Comment:
is a phone number needed in a 1:N call where all participants are an ACS participant?

Reply:
no its only needed for PSTN. I wonder though if we should have a note about how the calling modalities can mix so its encouraged so they can add a PSTN user to a 1:N call?

Reply:
Good point - phone numbers just add a whole nother level of complexity to get set up hoewever.
Regarding this doc line, can it be updated to something like `If you are looking to make a have PSTN attendees in the 1:N call then alongside the participantIDs you must also provide a [phone number ....`